routes/api.php

```diff
@@ -36,6 +36,8 @@
     });
 
     Route::group(['middleware' => 'custom.jwt.auth'], function () {
+        Route::get('/users/self/lists', 'Api\User\UserList\UserListController@userLists')
```

Comment:
'/users/me/lists' is more readable, we already have some routes with `me`, but in that case it should be separate action handler on backend like GetMyListsAction.php. So '/users/{user_id}/lists' is enough for now